yb-voyager/src/srcdb/srcdb.go

```diff
@@ -59,6 +59,7 @@ type SourceDB interface {
 	CheckSourceDBVersion() error
 	GetMissingExportSchemaPermissions() ([]string, error)
 	GetMissingExportDataPermissions(exportType string) ([]string, error)
+	CheckDependencies() error
```

Comment:
The more I think about it, the more I feel that this shouldn't be a method in sourceDB interface. 
RIght now, we have methods in sourceDB interface that do a lot more than just interact with the source DB. For example `ExportData` which essentially just calls pg_dump. Ideally, even `ExportData` shouldn't be part of SourceDB interface. It should be a method outside that takes a SourceDB as an input. 

Let me think about this more and get back.

Reply:
I think this function is better implemented outside pf sourceDB interface. 
We should only keep methods in sourceDB that directly interacts with the DB. This function is merely checking if all dependencies are met. In future, I think even ExportData should be moved out of SourceDB, since we are essentially using pg_dump/ora2pg to get data, not directly interacting with DB. If you look at how live migration/dbzm is implemented, the logic is defined in dbzm/exportData.go and it depends on SourceDB for getting config/metadata/etc. 

For now, I think you can define it in cmd pkg. 
This will also solve the circular dependency issue you were facing with calling code from dbzm pkg. 
cc: @priyanshi-yb  @sanyamsinghal

Reply:
Done. Added it to export.go. Added a check for DBZM too.